src/meltano/core/project_init_service.py

```diff
@@ -43,10 +43,19 @@ def init(self, activate: bool = True, add_discovery: bool = False) -> Project:
         """
         try:
             os.mkdir(self.project_name)
-        except Exception as exp:  # noqa: F841
+        except FileExistsError:
             raise ProjectInitServiceError(
                 f"Directory {self.project_name} already exists."
```

Comment:
This will add quotes around the project name in the output.

```suggestion
                f"Directory {self.project_name!r} already exists."
```